### DISCUSSION TITLE
Migrating some of my packages from fink

### DISCUSSION THREAD

**Discussion by wgscott** [Homebrew/discussions#958] [Everyday usage]
I maintain several packages in the sci category in fink, and I am beginning to realize that Homebrew might be a better option.  One in particular is a macromolecular graphics, crystallography/cryo-EM display and modeling software called Coot.  I've been working with the primary author for about the last 18 years maintaining it for OSX via fink, but have hit a brick wall due to lack of needed GTK+3 support.  I noticed that brewsci/bio has many of the same dependencies I maintain for fink, including one called clipper4coot, suggesting this is already on someone's wish list.  I've compiled more updated versions of these dependencies, and am using these along with Homebrew's GTK+3 to compile coot, so could potentially contribute those at some point, but would probably benefit from some guidance, especially to avoid duplication of effort (assuming that there is still interest).

Is there someone I should contact?

**Comment by SMillerDev** [comment]
You can make a pull request in brewsci/bio to add this software.

**Reply by gromgit** [reply]
Quick guide: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

**Comment by wgscott** [comment]
Thanks.

It isn't quite ready yet (I am sorting out some run-time errors) but wanted to find out if I am duplicating someone else's efforts.
